File: Lab-4/GCD.c
#include<stdio.h>
long int GCD(int a,int b)
{
    if(b==0)
        return a;
    else
        return GCD(b, a%b);
}
int main()
{
    int a,b;
    printf("Enter the values of a and b:");
    scanf("%d\n,%d",&a,&b);
    printf("The gcd of two number is:%d",GCD(a,b));
}

File: Lab-4/GCD.py
def gcd(a , b):
    if b==0:
        return a
    else:
        return gcd(b,a%b)

a = int(input("Enter the value of a: "))
b = int(input("Enter the value of b: "))
print(f"The GCD of two numbers is: {gcd(a, b)}")
